content/docs/reference/config/helm-customization/index.md

```diff
@@ -0,0 +1,283 @@
+| `kiali.image` | `` |  | `` |
+| `kiali.image.repository` | `kiali/kiali` |  | `` |
+| `kiali.image.tag` | `0.3.1.Alpha` |  | `` |
+| `kiali.ingress` | `` |  | `` |
+| `kiali.ingress.enabled` | `false` |  | `` |
+| `kiali.ingress.annotations` | `` |  | `` |
+| `kiali.ingress.tls` | `` |  | `` |
+| `kiali.resources` | `{}` |  | `` |
+| `kiali.dashboard` | `` |  | `` |
+| `kiali.dashboard.username` | `admin` |  | `` |
+| `kiali.dashboard.password` | `admin` |  | `` |
+<!-- AUTO-GENERATED-END -->
+
+The Helm chart also offers significant customization options per individual
+service. Customize these per-service options at your own risk. The per-service options are exposed via
+the [`values.yaml`]({{< github_file >}}/install/kubernetes/helm/istio/values.yaml) file.
+
+## Customization example: traffic management minimal set
+
+Istio is equipped with a rich and powerful set of features and some users may need only subset of those. For instance, users might be interested only
+in installing the minimal set required to Istio's traffic management.
+Helm customization provides the option to install a subset by enabling those features of interest and disabling the ones that aren't required.
+
+In this example we will install Istio with only a minimal set of components necessary to conduct [traffic management](/docs/tasks/traffic-management/).
+
+Execute the following command to install the Pilot, Citadel, IngressGateway and Sidecar-Injector:
+
+{{< text bash >}}
+$ helm install install/kubernetes/helm/istio --name istio --namespace istio-system \
+  --set ingress.enabled=false,gateways.istio-egressgateway.enabled=false,galley.enabled=false \
+  --set mixer.enabled=false,prometheus.enabled=false,global.proxy.envoyStatsd.enabled=false
+{{< /text >}}
+
+Ensure the following Kubernetes pods are deployed and their containers are up and running: `istio-pilot-*`, `istio-ingressgateway-*`,
+`istio-citadel-*` and `istio-sidecar-injector-*`.
+
+{{< text bash >}}
+$ kubectl get pods -n istio-system
+NAME                                     READY     STATUS    RESTARTS   AGE
+istio-citadel-b48446f79-wd4tk            1/1       Running   0          1m
+istio-ingressgateway-7b77d995f7-t6ssx    1/1       Running   0          1m
+istio-pilot-58c65f74bc-2f5xn             2/2       Running   0          1m
+istio-sidecar-injector-86cc99578-4t58m   1/1       Running   0          1m
+{{< /text >}}
+
+With this minimal set you can proceed to installing the sample [Bookinfo](/docs/examples/bookinfo/) application or install your own application and [configure request routing](/docs/tasks/traffic-management/request-routing/) for instance.
+
+Of course that if no ingress is expected and sidecar is to be [injected manually](/docs/setup/kubernetes/sidecar-injection/#manual-sidecar-injection) then you can reduce this minimal set even further and only have Pilot and Citadel. However, Pilot depends on Citadel therefore you can't install it without the other.
```

Comment:
If the minimal set can be further reduced, then it wasn't really the minimal set, right? Maybe you shold actually show the truly minimal set for traffic management?